### ISSUE TITLE
BinaryView.create_user_function() to return the created function

### ISSUE THREAD

**Issue by melomac** [Vector35/binaryninja-api#1318]
Here you have a practical example:

```diff
for addr, name in [(addr1, name1), (addr2, name2), (addrN, nameN)]:
    function = bv.get_function_at(addr)
    if not function and bv.is_valid_offset(addr):
-        bv.create_user_function(addr, bv.platform)
-        function = bv.get_function_at(addr)
+        function = bv.create_user_function(addr, bv.platform)
        if not function:
            continue
    function.name = name
```

**Comment by devtty1er** [comment]
+1 for this change and for similar changes across the API for all of the `define_*` and `create_* functions. I would PR
```

```

**Comment by KyleMiles** [comment]
Resolved in 2.3.2784-dev.
